### PR TITLE
Camel K 1.0.0

### PR DESCRIPTION
**Pull request by nicolaferraro**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by alebcay** [comment]
Dupe of #56021.

**Comment by ipolyzos** [comment]
> Dupe of #56021.

@alebcay, @SMillerDev  both PRs for updating kamel (#56021; and #56002)  are closed as duplicate to search other. Could you please reopen and merge?

**Comment by SMillerDev** [comment]
This PR is still open, there was just a mention it was duplicate

File: Formula/kamel.rb [hunk 1]
```diff
@@ -3,17 +3,10 @@ class Kamel < Formula
...
+    :revision => "38c24698b16da41926be5f7984115d428a825a02"
   head "https://github.com/apache/camel-k.git"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "1bca7ed6593fd50b219353b033b76195881576d5edce8154241edb3d249f665e" => :catalina
-    sha256 "349b58bd34eeb004baca62ee0e56746cb1256979c83f87aceb62882003b902a3" => :mojave
-    sha256 "7dc0dd18e8b3fa8ebaf322db37369929909510c65e8b5472a4624ee822771698" => :high_sierra
-  end
-
```
Comment (SMillerDev) [inline Formula/kamel.rb:16]: No need to remove the bottle block

Comment (nicolaferraro) [reply Formula/kamel.rb]: Thanks, restored

**Comment by SMillerDev** [comment]
Please squash all commits into one `camel-k 1.0.0`

**Comment by Rylan12** [comment]
Hey, sorry to butt in here when it looks like this is almost done.

If possible, could `Utils.popen_read` be switched to `Utils.safe_popen_read` for the completion installation lines? The safe version just serves as a check to make sure that the command it runs succeeds.

We're in the process of switching formulae to use the safe version, so if it's changed here that could kill two birds with one stone.

If not, no worries, we can fix it in the future.

**Comment by nicolaferraro** [comment]
> Hey, sorry to butt in here when it looks like this is almost done.
> 
> If possible, could `Utils.popen_read` be switched to `Utils.safe_popen_read` for the completion installation lines? The safe version just serves as a check to make sure that the command it runs succeeds.
> 
> We're in the process of switching formulae to use the safe version, so if it's changed here that could kill two birds with one stone.
> 
> If not, no worries, we can fix it in the future.

I think some commands return an error and we check the error message here. But I think we can do a simpler test to check correctness here. Let me work on it

**Comment by nicolaferraro** [comment]
> Hey, sorry to butt in here when it looks like this is almost done.
> 
> If possible, could `Utils.popen_read` be switched to `Utils.safe_popen_read` for the completion installation lines? The safe version just serves as a check to make sure that the command it runs succeeds.
> 
> We're in the process of switching formulae to use the safe version, so if it's changed here that could kill two birds with one stone.
> 
> If not, no worries, we can fix it in the future.

I should be fixed

**Review by SMillerDev — APPROVED** [review]
Thanks @nicolaferraro ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**